test/regress/cli/CMakeLists.txt

```diff
@@ -1501,6 +1505,9 @@ set(regress_0_tests
   regress0/quantifiers/clock-10.smt2
   regress0/quantifiers/clock-3.smt2
   regress0/quantifiers/cond-var-elim-binary.smt2
+  regress0/quantifiers/dd_check-cc_bvxor.smt2
+  regress0/quantifiers/dd_check-cc_bvxor-mpath.smt2
+  regress0/quantifiers/dd_check_eq_bvnot_32bit.smt2
```

Comment:
missing `dd_water_tank-node...`

Reply:
Actually it is a bit lower in the list...

Reply:
right sorry. but still there is a mismatch between the number of changed lines in this file and the number of new regressions in this PR.